docs/containers-conf.md

```diff
@@ -79,45 +79,132 @@ $ echo "MTAwMA==" | base64 --decode
     $ kubectl apply -f deploy/cr.yaml
     ```
 
-## Configure alternative memory allocator
+## Configure memory allocator
 
-You can use an alternative memory allocator library for `mysqld` to optimize memory usage. This is often recommended when memory usage is higher than expected.
+You can use an alternative memory allocator library for `mysqld` to optimize memory usage. This is often recommended when memory usage is higher than expected or when you need better memory fragmentation handling.
 
-The Percona XtraDB Cluster Pods include the jemalloc allocator. You can enable it with the `LD_PRELOAD` environment variable:
+Percona XtraDB Cluster supports two alternative memory allocators:
 
-```bash
-LD_PRELOAD=/usr/lib64/libjemalloc.so.1
+* **jemalloc** - A general-purpose memory allocator that emphasizes fragmentation avoidance and scalable concurrency support
+* **tcmalloc** - Google's memory allocator optimized for performance
+
+By default, Percona XtraDB Cluster uses the standard libc allocator. You can switch to an alternative allocator using the `mysqlAllocator` option in your Custom Resource.
+
+!!! note
+
+    The `mysqlAllocator` option is available only for Percona XtraDB Cluster 8.0 and later. For PXC 5.7, you must use the legacy method with environment variables. See the [Legacy method using environment variables](#legacy-method-using-environment-variables) section.
```

Comment:
this is not accurate. `mysqlAllocator` is available for 5.7 too and people can use it. we just don't have `jemalloc` or `tcmalloc` in official 5.7 images. so even if 5.7 users try to use the legacy method, it won't work.

Reply:
So 5.7 users have no way to use jemalloc, correct?